t/issues/t4482-flush-list-corruption.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh -e
+
+# How this test works
+#
+# use rc1 script that does not load content-sqlite
+#
+# add some unique data to the kvs, we do multiple puts to build up
+# a decent length internal list of flushable cache entries.
+#
+# drop kvs cache, this will force future KVS puts of identical data to
+# be sent to the content-cache
+#
+# write the same data again, if error present internal flush list will
+# be messed up and length of flush list < number of dirty entries
+# (acct_dirty).
+#
+# before fix, flux content flush will hang b/c number of dirty entries
+# (acct_dirty) never reaches zero.
+
+cat <<-EOF >t4482.sh
+#!/bin/sh -e
+
+flux kvs put issue4482A.a="abcdefghijk"
+flux kvs put issue4482A.b="lmnopqrstuv"
+flux kvs put issue4482A.c="wxyz0123456"
+flux kvs put issue4482A.d="7890ABCDEFG"
+flux kvs put issue4482A.e="HIJKLMNOPQR"
+flux kvs put issue4482A.f="STUVWXYZ!!!"
+flux kvs put issue4482A.g="<<<<<:>>>>>"
+
+flux kvs dropcache
+
+flux kvs put issue4482B.a="abcdefghijk"
+flux kvs put issue4482B.b="lmnopqrstuv"
+flux kvs put issue4482B.c="wxyz0123456"
+flux kvs put issue4482B.d="7890ABCDEFG"
+flux kvs put issue4482B.e="HIJKLMNOPQR"
+flux kvs put issue4482B.f="STUVWXYZ!!!"
```

Comment:
I'm not sure this is doing what it looks like at face value, although it still may work.  Those short values will be cached inside the directory entry for "issue4482B" so really what you're doing is creating multiple versions of that directory and the root in the content store.

Suggestion: use `flux content store` since the problem this pokes at has nothing to do with the kvs.

Reply:
ahhh yeah, you're right my description is not correct, but I the the effect is identical, the multiple versions of the directory are the "data", not the junk I'm writing.

Let me try with `flux content store`